app/konnect/getting-started/publish-service.md

```diff
@@ -46,27 +46,28 @@ descriptions.
 
     ```
 
-1. In the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), open a service.
-Find the **Service Document** section on your service's overview page and click **Upload Document**.
+1. In the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service.
+
+1. Select **Documentation**, then **Add Page**. Upload the `.md` file and **Save**.
+
+    {% konnect_icon servicehub %} may automatically populate fields from the uploaded documentation.
 
-1. Locate the `.md` file and click **Open**.
 
 ## Upload an API spec
 
-API specifications, or specs, can be uploaded and attached to a specific service version within your Dev Portal.
+OpenAPI specifications, or specs, can be uploaded and attached to a specific service version within your Dev Portal.
 Every version can have one OpenAPI spec associated with it, in JSON or YAML format.
 
-If you brought your own spec, use it in the following steps. Otherwise, you can
-use the [sample Vitals spec](/konnect/vitalsSpec.yaml) for testing.
+If you have a spec, use it in the following steps. Otherwise, you can
+use the [sample Analytics spec](/konnect/vitalsSpec.yaml) for testing.
```

Comment:
Could we also link to a page that describes how to create a spec? It seems like the options a customer would want here are:

- I have my own spec
- I need to create a spec
- I want to test a spec

Reply:
spoke offline no link needed